components/TextField/index.jsx

```diff
@@ -114,11 +116,31 @@ export default class TextField extends Component {
     this.setState({showError: false});
   }
 
+  renderInputLabel () {
+    const {
+      requiredLabel,
+      maxLength,
+      displayMaxLength
+    } = this.props;
+    const {value} = this.state;
+
+    // Show requiredLabel if available and user hasn't typed any values
+    if (requiredLabel && (!displayMaxLength || value.trim() === '')) {
+      return requiredLabel ? (<span className={`${controlNote} ${controlLabelRequired}`}>{requiredLabel}</span>) : null;
+    }
+    return displayMaxLength ? (
```

Comment:
as the output had format xx/xx I think this prop should be simpler like: `displayLength`